docs/breaking-changes.md

```diff
@@ -716,6 +716,18 @@ to open synchronously scriptable child windows, among other incompatibilities.
 See the documentation for [window.open in Electron](api/window-open.md)
 for more details.
 
+### Deprecated: `app.runningUnderRosettaTranslation`
+
+The `app.runningUnderRosettaTranslation` property has been deprecated.
+Use `app.runningUnderARM64Translation` instead.
```

Comment:
This is a very old version and people might miss this note since they upgraded a long time ago. Maybe an additional note should be added to the upcoming "removed" section? One that points out it was deprecated recently, the version it was deprecated from, etc.?